cluster/common.sh

```diff
@@ -44,11 +45,18 @@ function create-kubeconfig() {
   "${kubectl}" config set-cluster "${CONTEXT}" --server="https://${KUBE_MASTER_IP}" \
                                                --certificate-authority="${CA_CERT}" \
                                                --embed-certs=true
-  "${kubectl}" config set-credentials "${CONTEXT}" --username="${KUBE_USER}" \
-                                                --password="${KUBE_PASSWORD}" \
-                                                --client-certificate="${KUBE_CERT}" \
-                                                --client-key="${KUBE_KEY}" \
-                                                --embed-certs=true
+  if [[ -z "${KUBE_USER:-}" || -z "${KUBE_PASSWORD:-}" ]]; then
```

Comment:
What would you recommend doing if it isn't? The previous code just assumes that all of the necessary vars are set, and after a transition period I'd like to remove the username/password option from all cloud providers and transition everyone to using bearer tokens. I suppose we could skip writing any credentials to the kubeconfig file if they aren't set, but that means there is a bug in the startup scripts and the user won't be able to interact with their cluster.